cernopendata/modules/fixtures/data/records/opera-author-list-tau.json

```diff
@@ -554,7 +554,19 @@
     ],
     "date_created": "2010-2012",
     "date_published": "2018",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ]
+    },
     "experiment": "OPERA",
+    "files": [
+      {
+        "checksum": "adler32:0cf746ba",
+        "size": "143315",
```

Comment:
Note that `size` is an integer, not a string... Can you please amend and repush?